Version 1.24.8 of grpc and updated node-pre-gyp name

### Problem description

```Cannot find module 'node-pre-gyp/lib/pre-binding'```

The commit [here](https://github.com/grpc/grpc-node/commit/c6525d2ab0f7962638d539ea59523f13d6502a63) updates the [installed dep to `@mapbox/node-pre-gyp`](https://github.com/grpc/grpc-node/blob/c6525d2ab0f7962638d539ea59523f13d6502a63/packages/grpc-native-core/package.json#L29) but forgot to update the `require` statement [here](https://github.com/grpc/grpc-node/blob/c6525d2ab0f7962638d539ea59523f13d6502a63/packages/grpc-native-core/src/grpc_extension.js#L26).  

### Reproduction steps

n/a

### Environment

n/a

Yes for me too. Failing after installing Version 1.24.8 and
`const grpc = require("grpc") `
results in
Cannot find module 'node-pre-gyp/lib/pre-binding'

--- a/packages/grpc-native-core/binding.gyp
+++ b/packages/grpc-native-core/binding.gyp
@@ -91,7 +91,7 @@
       'GPR_BACKWARDS_COMPATIBILITY_MODE',
       'GRPC_ARES=1',
       'GRPC_UV',
-      'GRPC_NODE_VERSION="1.24.8"',
+      'GRPC_NODE_VERSION="1.24.9"',
       'CARES_STATICLIB',
       'CARES_SYMBOL_HIDING'
     ],

--- a/packages/grpc-native-core/build.yaml
+++ b/packages/grpc-native-core/build.yaml
@@ -1,3 +1,3 @@
 settings:
   '#': It's possible to have node_version here as a key to override the core's version.
-  node_version: 1.24.8
+  node_version: 1.24.9

--- a/packages/grpc-native-core/package.json
+++ b/packages/grpc-native-core/package.json
@@ -1,6 +1,6 @@
 {
   "name": "grpc",
-  "version": "1.24.8",
+  "version": "1.24.9",
   "author": "Google Inc.",
   "description": "gRPC Library for Node",
   "homepage": "https://grpc.io/",

--- a/packages/grpc-native-core/src/grpc_extension.js
+++ b/packages/grpc-native-core/src/grpc_extension.js
@@ -23,7 +23,7 @@
 
 'use strict';
 
-var binary = require('node-pre-gyp/lib/pre-binding');
+var binary = require('@mapbox/node-pre-gyp/lib/pre-binding');
 var path = require('path');
 var binding_path =
     binary.find(path.resolve(path.join(__dirname, '../package.json')));

